schemaless/sources.py

```diff
@@ -382,6 +382,7 @@ class Planning(DirectSource):
         'blocklot': Concat('block', 'lot'),
     }
     DATA_SF = "https://data.sfgov.org/dataset/PPTS-Records_data/kgai-svwy"
+    DATA_SF_DOWNLOAD = "https://data.sfgov.org/api/views/kgai-svwy/rows.csv?accessType=DOWNLOAD"  # NOQA
```

Comment:
Isn't this not public right now? How does this work but the other ones don't?

Reply:
This Source is no longer used, actually. I should delete it.

Reply:
Edit: my mistake. I actually put the wrong URL here! The Source is right

Reply:
fixed